legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -425,7 +425,7 @@ def display_name(business: any, filing: FilingStorage) -> Optional[str]:
 
         business_revision = business
         # retrieve business revision at time of filing so legal type is correct when returned for display name
-        if bus_rev_temp := VersionService.get_business_revision_obj(filing, business):
+        if bus_rev_temp := VersionService.get_business_revision_obj(filing, business.id):
```

Comment:
so this function was updated to take in a business identifier instead of a business object.  think it should be `business.identifer`?